xgboost_ames_housing/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+  - jlewi
+  - richardsliu
+  - zhenghuiwang
+reviewers:
```

Comment:
nit: You don't need to list me here; I'm already in the parent file and I don't have any special familiarity with the xgboost example.

don't list approvers as reviewers; all approvers are automatically considered reviewers.

Reply:
I should have read the Owner guideline more carefully. My fault... it should be fine this time.